sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1572,8 +1572,8 @@ static int sof_card_dai_links_create(struct snd_soc_card *card)
 	int num_links, link_index = 0;
 	char *name, *cpu_dai_name;
 	char *codec_name, *codec_dai_name;
+	int codec_index, hdmi_num;
```

Comment:
i really don't like two variables without any correlation on the same line. It's legal in C but it looks weird. Why would we group codec_index and hdmi_num?

Reply:
OK